src/site/content/en/handbook/web-dev-components/index.md

```diff
@@ -161,6 +162,56 @@ Use the codelab aside to link to an associated codelab.
   [Using Imagemin with Grunt](#)
 {% endAside %}
 
+## Author
```

Comment:
Please remove. This component is not intended to be used by authors inside prose.